posts/2023-10-17-translation_announcement-toby_hocking/index.qmd

```diff
@@ -27,14 +32,20 @@ Over the course of the NSF project (Sep 2023 to Aug 2025), there will be 20 tran
 
 Applications should be submitted to **toby.hocking\@r-project.org**, and should include the following information:
 
--   what documents/materials do you propose to translate? (messages/vignettes/other)
+-   who is the project leader? Projects which include two or more people are encouraged, in order to proof-read translations, and ensure quality. The project leader should be a trusted member of the R community, who will be responsible for proof-reading and approving other project members' translations.
+-   how many people will be translating, and how many US\$500 awards are you requesting? 
 -   which non-English language?
--   how many people will be translating, and how many US\$500 awards are you requesting? Projects which include two or more people are encouraged, in order to proof-read translations, and ensure quality.
--   for each person in your project, what level of fluency in English and other languages?
--   for each person in your project, how many years using R/data.table?
--   how will your translation project help create an expanded ecosystem of users and contributors?
+-   for each person in your project, what level of fluency in English
+    and other languages? for each person in your project, what local
```

Comment:
Regarding dialects, maybe it's better to focus the questionon the team. Something like, "how many dialects are represented on your team? (e.g., spanish, mexican, chilean Spanish, etc)

Reply:
great thanks for the suggestion